android/WALT/app/src/main/java/org/chromium/latency/walt/FrontPageFragment.java

```diff
@@ -42,7 +41,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         View view = inflater.inflate(R.layout.fragment_front_page, container, false);
 
         if (container.getContext().getPackageManager().
-                hasSystemFeature(PackageManager.FEATURE_MIDI)) {
+                hasSystemFeature("android.software.midi")) {
             view.findViewById(R.id.action_midi).setVisibility(View.VISIBLE);
```

Comment:
That sounds good. I changed this because the linter was complaining: the string constant is only available in newer API levels, so I hardcoded the string instead. In the future we should upgrade the minSdk to 23.